iina/MPVController.swift

```diff
@@ -827,6 +828,9 @@ class MPVController: NSObject {
         player.sendOSD(.hwdec(player.info.hwdecEnabled))
       }
 
+    case MPVOption.Video.videoRotate:
+      player.info.rotation = Int(getInt(MPVOption.Video.videoRotate))
```

Comment:
This is getInt method that is being called:
```swift  
func getInt(_ name: String) -> Int {
    var data = Int64()
    mpv_get_property(mpv, name, MPV_FORMAT_INT64, &data)
    return Int(data)
  }
```
So I'm not understanding the need to nest the call to getInt inside "Int(...)"? This is the same pattern used in the other calls to `getInt`, so good to be consistent, no need to update this commit. Possibly something to eventually clean up. Or am I confused about Swift?